components/Table/TableKDA.css

```diff
@@ -0,0 +1,19 @@
+@import '../palette.css';
+@import "./TablePercent.css";
+
+:export {
```

Comment:
Yea I agree. I remember at one point webpack had a bug where if you imported the same css file multiple times, it would duplicate it, but it doesn't seem to be the case anymore. My bad!